patch/kconfig-exclusions

```diff
@@ -15,6 +15,7 @@ CONFIG_MTD_SPI_NOR_USE_4K_SECTORS
 # Unset X86_PAT according to Broadcom's requirement
 CONFIG_X86_PAT
 CONFIG_MLXSW_PCI
+CONFIG_IMA
```

Comment:
Will that affect other systems/users?

Reply:
actually, I am not sure about this. We need other vendor to review this change.